packages/babel-jest/src/index.js

```diff
@@ -82,24 +79,20 @@ const createTransformer = (options: any) => {
       fileData: string,
       filename: Path,
       configString: string,
-      {instrument, watch}: TransformOptions,
+      {instrument}: TransformOptions,
     ): string {
-      return (
-        crypto
-          .createHash('md5')
-          .update(THIS_FILE)
-          .update('\0', 'utf8')
-          .update(fileData)
-          .update('\0', 'utf8')
-          .update(configString)
-          .update('\0', 'utf8')
-          // Don't use the in-memory cache in watch mode because the .babelrc
-          // file may be modified.
-          .update(getBabelRC(filename, {useCache: !watch}))
```

Comment:
That's the reason I rewrote transform into ScriptTransformer. What we do now when we get a transformer, we check if `createTransformer` exists. If it does, we call that. We do this now once per Runtime instance (once per Test). This means every test will have its own babelrc file cache instead of having a single cache for the entirety of Jest's lifetime. Does that make sense?